### PR TITLE
Make codekit work with both Python 2 and 3: Tickets/dm 8388

### PR DESCRIPTION
**Pull request by athornton**
Make Codekit work with both Python 2 and 3

### CODE REVIEW

**Comment by jhoblitt** [comment]
It would be nice to add py3 to the travis matrix as part of the PR.

File: codekit/cli/github_delete_shadow.py [hunk 1]
```diff
@@ -50,47 +54,47 @@ def main():
     orgname = '{user}-shadow'.format(user=args.user)
 
     if args.debug:
-        print 'org:', orgname
+        print('org:', orgname)
 
-    gh = codetools.login_github(token_path=args.token_path)
+    ghb = codetools.login_github(token_path=args.token_path)
```
Comment (jhoblitt) [inline codekit/cli/github_delete_shadow.py]: the linter doesn't like 2 char var names?

File: codekit/licensing.py [hunk 2]
```diff
@@ -162,6 +164,8 @@ def upgrade_repo(gh, github_repo, branch_name, clone_dir):
     clone_dir : str
         Directory to clone the repository into.
     """
+    # gh is not actually used.
+    gh = gh
```
Comment (jhoblitt) [inline codekit/licensing.py]: Why not remove the declaration?

Comment (athornton) [inline codekit/licensing.py]: Because I don't know who's using the function, and changing the prototype will break calls.

Comment (jhoblitt) [inline codekit/licensing.py]: fair enough.

**Comment by jhoblitt** [comment]
👍

File: codekit/licensing.py (hunk 2 above)
Comment (jonathansick) [inline codekit/licensing.py]: Does a python linter require that all arguments are used??? I don't get this.

**Pull request merged**